examples/stateful-counter/README.md

```diff
@@ -0,0 +1,210 @@
+## Example
+
+We will create a simple stateful smart contract which will:
+- Have a global variable `counter`
+- Will increment `counter` each time we call the application.
+
+## Steps
+
++ Create a file with Approval Program
```

Comment:
```suggestion
+ [Create a file with Approval Program](https://github.com/scale-it/algo-builder/blob/1307db9b620d8715c844bdf5c329599e557aade8/examples/stateful-counter/README.md#1-create-a-file-with-approval-program)
```
Can we hyperlink these points to their description below as well (as suggested above) ? note: probably the link is not right

Reply:
yes